experiments/chaos/openebs_target_failure/chaosutil.j2

```diff
@@ -7,12 +7,10 @@
     {% endif %}
   {% endif %}
 {% endif %}
-{% if stg_prov is defined and (stg_prov == 'openebs.io/provisioner-iscsi' or stg_prov == 'jiva.csi.openebs.io') %}
-  {% if stg_engine is defined and stg_engine == 'jiva' %}
+{% if stg_prov is defined and (stg_prov == 'jiva.csi.openebs.io') %}
```

Comment:
@payes  In this chaos template the condition to call jiva related utils are removed?

Reply:
added back, thanks.